test/nim/collectibles_types_conversions_test.nim

```diff
@@ -0,0 +1,43 @@
+import unittest
+
+import stint
+import backend/collectibles_types
+import app_service/service/community_tokens/community_collectible_owner
+include app_service/common/json_utils
+
+suite "collectibles types":
+
+  test "CollectibleOwner json conversion":
+
+    let oldBalance1 = CollectibleBalance(tokenId: stint.u256(23), balance: stint.u256(41))
+    let oldBalance2 = CollectibleBalance(tokenId: stint.u256(24), balance: stint.u256(42))
```

Comment:
I would add some really big value here from string to check big ints handling as well.